tools/liquibase/generator.go

```diff
@@ -0,0 +1,80 @@
+package liquibase
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"html/template"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"time"
+)
+
+type Generator struct{}
+
+func (g Generator) Name() string {
+	return "liquibase"
+}
+
+func (g Generator) Generate(ctx context.Context, root string, args []string) error {
+
+	t := time.Now()
+	fecha := fmt.Sprintf("%d%02d%02d%02d%02d%02d", t.Year(), t.Month(), t.Day(), t.Hour(), t.Minute(), t.Second())
+
+	name := args[3]
+	var underscoreName string
+	re := regexp.MustCompile(`[a-zA-Z][^A-Z]*`)
+	submatchall := re.FindAllString(name, -1)
+
+	for index, element := range submatchall {
+		if index == 0 {
+			underscoreName = strings.ToLower(element)
+			continue
+		}
+		underscoreName = underscoreName + "_" + strings.ToLower(element)
+	}
+	fullName := fecha + "-" + underscoreName + ".xml"
```

Comment:
@YanDeLeon this one should be done by an inflection library, see https://github.com/gobuffalo/flect/blob/master/underscore.go#L12